llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -702,14 +751,6 @@ def tensor_hook(
                         True,
                     ) if is_te_imported and state.precision == Precision.AMP_FP8 else contextlib.nullcontext(
                     )
-                    with context_manager:
```

Comment:
not sure if an equivalent to this is necessary or not, can you determine that?

Reply:
I included this outside the if statement. Presumably if this was necessary, I don't want to modify the existing logic

Reply:
Saw this in the logs 
```
2024/10/26 01:21:26 INFO mlflow.transformers: A local checkpoint path or PEFT model is given as the `transformers_model`. To avoid loading the full model into memory, we don't infer the pip requirement for the model. Instead, we will use the default requirements, but it may not capture all required pip libraries for the model. Consider providing the pip requirements explicitly.
```
so we don't need this code